crates/core/src/db/queries/feed_event.rs

```diff
@@ -25,14 +25,16 @@ use crate::{
 /// join of event tables into a single event type
 pub type Columns<'a> = (
     models::FeedEvent,
+    String,
+    Option<String>,
```

Comment:
What are the column names for these two types?  Some descriptive comments like some of the other query modules have would be good